workflow-templates/im-test-k6-manual-advanced.yml

```diff
@@ -0,0 +1,66 @@
+name: Manual trigger K6 test - Advanced
+
+on: 
+  workflow_dispatch:
+    inputs:
+      envToTest:
+        description: 'Environment to run test against [[1] dev, [2] qa, [3] stage]'   
```

Comment:
This should be changed to use the name of the environment.

Reply:
Updated.